resources/systems/linux/es_find_rules.xml

```diff
@@ -810,10 +810,11 @@
             <entry>ruffle</entry>
         </rule>
         <rule type="staticpath">
+            <entry>/app/bin/ruffle</entry>
```

Comment:
You said you're using a wrapper, is this the wrapper? Otherwise it will call the original Ruffle.
In case the wrapper should be recognizable, I usually use a naming convention: emulator-rd-wrapper, in this case ruffle-rd-wrapper

Reply:
I think I looked at what was done for Pico 8 and followed that example. Ie copy the wrapper to a new name in the manifest.

Following this suggestion is what I would have done if I had looked at Pico 8.

I'll change the es-de pr and yml in feat/ruffle.

I'll also pick a version of ruffle to use. Then I can remove it from the placeholder.